Alignment/OfflineValidation/python/TkAlAllInOneTool/alignment.py

```diff
@@ -7,9 +7,10 @@ class Alignment:
     def __init__(self, name, config, runGeomComp = "1"):
         self.condShorts = {
             "TrackerAlignmentErrorExtendedRcd":
-                {"zeroAPE":{"connectString": ("frontier://FrontierProd"
-                                              "/CMS_COND_31X_FROM21X"),
-                            "tagName": "TrackerIdealGeometryErrors210_mc",
+                {"zeroAPE":{"connectString": ("frontier://PromptProd"
```

Comment:
why PromptProd? It cannot be used for tests as it is the Tier-0 servlet. Please use FrontierProd